Sem. 04/Pract.04/week4_pract_Task12.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+int secondLength(unsigned int second){
+    int len=0;
+    while(second!=0){
+        len++;
+        second /= 10;
+    }
+    return len;
+}
+int pow(int ten , int len){
```

Comment:
Here as well - the parameters' names must be more generic - base and exponent for example. You will need the same function later and the base won't be 10.